tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: push-snapshot
   labels:
-    app.kubernetes.io/version: "1.0.1"
+    app.kubernetes.io/version: "1.0.2"
```

Comment:
This is more than just a bug fix, I'd go to 1.1.0

Reply:
Done!